environment.yml

```
@@ -16,7 +16,7 @@ dependencies:
   - pint
   - pip
   - pip:
-      - git+https://github.com/libra-project/libra-toolbox # add version tag
+      - git+https://github.com/libra-project/libra-toolbox@8644e0a62ebbbcc5def3401954bc8ab1ff91e734 # add version tag
```

```suggestion
      - git+https://github.com/libra-project/libra-toolbox@0.8
```

We have released all of this to a new release

---

Nevermind, fixed with [2c53d09](https://github.com/LIBRA-project/BABY-1L-run-6/pull/5/commits/2c53d0920661574adb1266072c6d12910df9bb06)